spec/v1.0-draft/digest_set.md

```diff
@@ -18,23 +18,24 @@ metadata object.
 ## Fields
 
 A DigestSet is represented as a _JSON object_ mapping algorithm name to
-lowercase hex-encoded value. Usually there is just a single key/value pair,
-but multiple entries MAY be used for algorithm agility.
-
-> **TODO**: Add language for supporting base64-encoded digest values (re: goModuleH1).
+a string encoding of the digest using that algorithm. The named standard
+algorithms below use lowercase hex encoding. Usually there is just a
+single key/value pair, but multiple entries MAY be used for algorithm
+agility.
 
 Supported algorithms:
 
--   Standard cryptographic hash algorithms, for cases when the method
-    of serialization is obvious or well known:
+-   Standard cryptographic hash algorithms with lowercase hex-encoded
+    values, for cases when the method of serialization is obvious or
+    well known:
```

Comment:
One more thing. Might be good to clarify the naming of the algorithms per https://github.com/in-toto/attestation/pull/135#discussion_r1114675451

Reply:
Done!